model-engine/model_engine_server/inference/batch_inference/vllm_batch.py

```diff
@@ -30,6 +32,8 @@
 
 os.environ["AWS_PROFILE"] = os.getenv("S3_WRITE_AWS_PROFILE", "default")
 
+CPU_COUNT = multiprocessing.cpu_count()
```

Comment:
I remember that there may be an issue with this, where this returns the # of CPUs on the host, as opposed the # of CPUs allotted to the pod in k8s.

Reply:
good point. let me dig a bit

Reply:
use more accurate calculations